huggan/pytorch/lightweight_gan/cli.py

```diff
@@ -100,7 +101,16 @@ def train_from_folder(
     mixed_precision = "no",
     show_progress = False,
     wandb = False,
+    push_to_hub = False,
+    organization_name = None,
 ):
+    if push_to_hub:
+        if name == 'default':
+            raise RuntimeError(
+                "You've chosen to push to hub, but have left the --name flag as 'default'."
+                " You should name your model something other than 'default'!"
+            )
```

Comment:
Should this also raise an error if no `organization_name` is specified?

In the other scripts, this defaults to "huggan"